yarn.lock

```diff
@@ -9123,7 +9364,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"vite@npm:^5.0.0 || ^6.0.0, vite@npm:^6.0.0":
+"vite@npm:^5.0.0 || ^6.0.0":
   version: 6.3.3
   resolution: "vite@npm:6.3.3"
   dependencies:
```

Comment:
It still have vite v6 in here, I think either you would need to remove the ^ in the package.json so its exact, install then put it back, or maybe `yarn dedupe` will remove it.